.github/workflows/backend.yml

```diff
@@ -831,7 +831,7 @@ jobs:
 
   jdbc-connectors-it-part-1:
     needs: [ changes, sanity-check ]
-    if: needs.changes.outputs.api == 'true'
+    if: needs.changes.outputs.api == 'true' || contains(needs.changes.outputs.it-modules, 'connector-jdbc-e2e-part-1')
```

Comment:
please revert ci config, because if `needs.changes.outputs.api == 'false'`, the jdbc test case would run in `updated-modules-integration-test-part`

Reply:
> please revert ci config, because if `needs.changes.outputs.api == 'false'`, the jdbc test case would run in `updated-modules-integration-test-part`

ok